webextensions/api/tabs.json

```
@@ -2184,7 +2184,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/getZoom",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "38"
```

:x: https://github.com/chromium/chromium/commit/63d1f9b9a1fb0e67739fdfe59d4aa5a978bf95ac implemented in [38.0.2084.0](https://github.com/chromium/chromium/releases/tag/38.0.2084.0) before 38.0.2125, but only https://github.com/chromium/chromium/commit/255e72284868d9d234b428abe2d241d637905bbd in [42.0.2282.0](https://github.com/chromium/chromium/releases/tag/42.0.2282.0) before 42.0.2311 shipped.

```suggestion
                "version_added": "42"
```